### PR TITLE
addition of seeder addon

### PR DESCRIPTION
**Pull request by ibrokethecloud**
**Problem:**

PR introduces harvester-seeder addon packaging in installer.

**Solution:**

**Related Issue:** https://github.com/harvester/harvester/issues/2318

**Test plan:**

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM!

File: scripts/build-bundle [hunk 1]
```diff
@@ -159,6 +165,9 @@ echo ${VM_IMPORT_CONTROLLER_IMAGE} >>  ${image_list_file}
 # pcidevices-controller images
 echo ${PCIDEVICES_CONTROLLER_IMAGE} >> ${image_list_file}
 
+# pcidevices-controller images
```
Comment (bk201) [inline scripts/build-bundle:168]:
```suggestion
# seeder images
```

**Review by bk201 — APPROVED** [review]

**Comment by guangbochen** [comment]
@mergify backport v1.2

**Pull request merged**